config/routes.rb

```diff
@@ -56,7 +56,7 @@
       end
     end
 
-    resources :releases, only: [:show, :index, :new, :create]
+    resources :releases, only: [:show, :index, :new, :create], id: /v(\d+(:?\.\d+)*)/, format: /json/
```

Comment:
I can't use the `VERSION_REGEX` because routes don't allow string anchors, but I can create a new one without them.  Also do we need `no format`?  It defaults to JSON, right?